Sources/HTTPTypesFoundation/URLSession+HTTPTypes.swift

```diff
@@ -113,7 +113,14 @@ extension URLSession {
         }
         return (location, response)
     }
+}
+
+#endif
+
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS) || os(visionOS)
```

Comment:
Can we guard only the `bytes` func with this directive? That way we don't have to do the `#endif/#if` and split the extension. LGTM otherwise!

Reply:
I put it all together in one extension.